invenio_app_ils/ui/src/invenio_app_ils/authentication/components/AuthControlWrapper/AuthControlWrapper.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { sessionManager } from '../../../authentication/services';
+import { LoginRedirectButton } from '../LoginRedirectButton/LoginRedirectButton';
+
+export class AuthControlWrapper extends Component {
```

Comment:
I saw this one in another PR no?

Reply:
I think something went wrong with my rebase, sorry!